### DISCUSSION TITLE
［求助］这是哪里?

### DISCUSSION THREAD

**Discussion by Gftp3** [share121/inter-knot#1656] [其他]
我刚下列车，手里就突然多出了这么一个小玩意儿。而且谁能告诉我这是哪？
![IMG_20240812_134952](https://github.com/user-attachments/assets/64e3bdaf-de05-4dc7-97df-9827b18cc973)

**Comment by Gftp3** [comment]
![Uploading IMG_20240812_134952.jpg…]()

**Reply by Myitian** [reply]
不要在图片没上传完成的时候发送啊喂

**Reply by Gftp3** [reply]
我不知道啊😖，我看他要加载完了，结果突然没信号了。刚刚才有一点信号，我才能回复你，但是现在我ha

**Comment by fu-fu404** [comment]
楼主丸辣，这里是零号空洞最里面，据说传奇绳匠【双子】进入到现在还没回来
